packages/ui/src/components/box/hstack.native.tsx

```diff
@@ -0,0 +1,26 @@
+import { createBox } from '@shopify/restyle';
+
+import { type Theme } from '../../theme-native';
+
+export const HStack = createBox<Theme>();
+
+HStack.defaultProps = {
+  flex: 1,
+  flexDirection: 'row',
+  alignItems: 'center',
+};
+
```

Comment:
I left this commented code as I'm not sure what the best way of doing this is. On web we often extend for example:
```
interface PeteBox extends BoxProps {
...
```

This way of re-creating custom boxes seems to work but I am curious as to whether it's best to do that or a variation of the commented code below